src/components/List/src/listitem.module.css

```diff
@@ -0,0 +1,78 @@
+.denhaag-list__item {
+  padding-block: var(--denhaag-listitem-padding-block);
+  padding-inline: var(--denhaag-listitem-padding-inline);
+}
+
+.denhaag-list__item:focus {
+  border-color: var(--denhaag-listitem-focus-border-color);
+  border-style: var(--denhaag-listitem-focus-border-style);
+  border-width: var(--denhaag-listitem-border-size);
+  padding-block: calc(var(--denhaag-listitem-padding-block) - var(--denhaag-listitem-border-size));
+  padding-inline: calc(var(--denhaag-listitem-padding-inline) - var(--denhaag-listitem-border-size));
+}
+
+.denhaag-list__item:active {
+  border: var(--denhaag-listitem-border-size) solid var(--denhaag-listitem-active-border-color);
+  padding-block: calc(var(--denhaag-listitem-padding-block) - var(--denhaag-listitem-border-size));
+  padding-inline: calc(var(--denhaag-listitem-padding-inline) - var(--denhaag-listitem-border-size));
+}
+
+.denhaag-list__item:active > .denhaag-list__secondaryaction {
+  right: calc(var(--denhaag-listitem-secondaryaction-right) - var(--denhaag-listitem-border-size));
+}
+
+.denhaag-list__item:focus > .denhaag-list__secondaryaction {
+  right: calc(var(--denhaag-listitem-secondaryaction-right) - var(--denhaag-listitem-border-size));
+  color: var(--denhaag-listitem-secondaryaction-focus-color);
+}
+
+.denhaag-list__item:focus .denhaag-list__itemtextprimary {
+  color: var(--denhaag-listitem-focus-color);
+}
+
+.denhaag-list__item:focus .denhaag-list__itemicon {
+  color: var(--denhaag-listitem-icon-focus-color);
+}
+
+.denhaag-list__item:hover > .denhaag-list__secondaryaction {
+  color: var(--denhaag-listitem-secondaryaction-focus-color);
+}
+
+.denhaag-list__item:hover .denhaag-list__itemtextprimary {
+  color: var(--denhaag-listitemtext-primarytext-hover-color);
+}
+
+.denhaag-list__item:hover .denhaag-list__itemicon {
+  color: var(--denhaag-listitem-icon-hover-color);
+}
+
+.denhaag-list__item[secondarytext] .denhaag-list__itemtextprimary {
```

Comment:
Does the attribute `secondary-text` actually exist, I can't seem to find it anywhere.
Furthermore, I would prefer using a modifier in this scenario, for example `.denhaag-list__item--multi-line`

Reply:
Good find, I forgot to rename the secondaryText property to secondarytext so css did not find it.
Solved a bug 👍!